try:
    l = ['ZeroDivisionError', 'NameError', 'IndentationError', 'IOError', 'EOFError', 'IndexError']
    print(l[0])
    print(l[6])
except:
    print('Es gibt einen Fehler im Code!')